plugins/tekton/src/utils/tekton-utils.ts

```diff
@@ -164,8 +194,8 @@ export const calculateDuration = (
 };
 
 export const pipelineRunDuration = (run: PipelineRunKind): string => {
-  const startTime = run?.status?.startTime;
-  const completionTime = run?.status?.completionTime;
+  const startTime = run.status?.startTime;
+  const completionTime = run.status?.completionTime;
```

Comment:
nit: do we need this, run is required but good to have a check to avoid NPE so can make check and do early return as well